plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -325,6 +328,8 @@ fu_plugin_thunderbolt_add (FuPlugin *plugin, GUdevDevice *device)
 		fu_device_set_version (dev, version);
 	if (is_host)
 		fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_INTERNAL);
+	fu_plugin_add_report_metadata (plugin, "ThunderboltSafeMode",
+				       is_safemode ? "True" : "False");
```

Comment:
1. Isn't a better place for it will be closer to the set of `is_safemode`?
1. Maybe better to add it only `if (is_host)`?

Reply:
Thanks for the comments, adjusted.